### ISSUE TITLE
Click method is not called on chrome android

### ISSUE THREAD

**Issue by HarelM** [angular/angular#17039]
**I'm submitting a ...**  (check one with "x")
```
[x] bug report => search github for a similar issue or PR before submitting (It's a bit hard with all the open issues but I did my best)
```

**Current behavior**
Click is not triggered on chrome android.

**Expected behavior**
click function should be called.

**Minimal reproduction of the problem with instructions**
Couldn't create a minimal reproduction plunker.
In the following site click on the chart icon in the right bottom corner - doing it from the PC presents a chart, when debugging from web ide for chrome the click is not triggered.
http://israelhiking.osm.org.il/test/
No errors in the console.
Not sure if this is a chrome android bug or an angular 4 bug, but it used to work with angularJS
(here: http://israelhiking.osm.org.il/)

**Please tell us about your environment:**
Win 10, visual studio, angular-cli.

* **Angular version:** 4.1.3

* **Browser:** 
Mobile Chrome latest 
 
* **Language:** all

* **Node (for AoT issues):** `node --version` =   6.10.3

**Comment by Toxicable** [comment]
Im sorry but I personally am not going to debug your application, please provide a minimal reproduction repo or plunkr. There is far too many variables when looking a specific issue thats in a whole project to be able to debug it effectively.
However I would advise that you check that you have all browser polyfills that you need: https://angular.io/docs/ts/latest/guide/browser-support.html

**Comment by HarelM** [comment]
I'll check if I can create a minimal reproduction although my initial attempts failed. I'd be happy to try and debug it, but I don't know where to start.
I'll check the polyfills although I guess it should show an error when attaching it with web IDE and I didn't see any errors.

**Comment by HarelM** [comment]
Please let me know if this is simple enough:
Issue can be seen here by clicking the the empty box at the top of the "tower" in the bottom right of the screen:
![image](https://cloud.githubusercontent.com/assets/3269297/26609946/2cb9bc72-45ad-11e7-97d2-88e873d7e47d.png)
When clicking it should open a popup - it doesn't happen on **mobile chrome*.
There are no polyfills required according to @Toxicable link as far as I can understand.
Here is the `ng build` outcome on my server:
http://israelhiking.osm.org.il/issue17039/
Sources can be found here:
[Issue17039-src.zip](https://github.com/angular/angular/files/1039948/Issue17039-src.zip)
I don't know how to debug this, the break point doesn't hit- it should hit toggle:
`RouteStatisticsComponent.prototype.toggle` but it doesn't on mobile chrome.

Please help me, I have no clue how to solve this and this is the only issue that prevents me from publishing my site.

I have constructed my code according to the example provided in issue #16582.

**Comment by HarelM** [comment]
@trotyl, any change for your feedback on this? The code is based on your suggestions and I'm not entirely sure I implemented them correctly.
Having said that, it does work on desktop and mobile FF so the code I wrote might not be the real issue here...

**Comment by HarelM** [comment]
Anyone???

**Comment by HarelM** [comment]
It seems this issue is resolved in a later versions of angular-cli or angular, I'm not sure, but I can't reproduce it anymore, so I'm closing. Thanks I guess :-)

**Comment by gauravsaluja2006** [comment]
Well, guess this is nearly an year after this was closed. I faced the same thing. 
Sometimes click works, sometimes it doesn't. 
Even more weird, 
1. Click not working on some part of my app
2. I click the home button on my android device. 
3. Tap on the chrome and come back to where i left before
4. Click works now suddenly
Updating CLI again, and building it seemed to fix it after everything.
